importantTest1.1/task2/main.c

```diff
@@ -0,0 +1,14 @@
+#include "priorityQueue.h"
+
+#include <stdio.h>
+
+int main()
+{
+    PriorityQueue* queue = priorityQueueCreate();
+    priorityQueueEnqueue(1, 1, queue);
+    priorityQueueEnqueue(5, 5, queue);
+    priorityQueueEnqueue(3, 3, queue);
+    priorityQueueEnqueue(4, 1, queue);
+    priorityQueueFree(queue);
```

Comment:
```suggestion
    priorityQueueFree(&queue);
```
Она же принимает двойной указатель. Если ей передать одинарный, она его интерпретирует как двойной, пытается удалить не то, и программа падает.